docs/list.md

```diff
@@ -251,6 +262,17 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **Issues**: [learn-graphql issues](https://github.com/hasura/learn-graphql/issues) and [graphql-engine issues](https://github.com/hasura/graphql-engine/issues).
 - **Notes**: For more information, check out [this blog post](https://hasura.io/blog/hasura-joins-hacktoberfest-3rd-year-in-a-row/) and [this blog post too](https://hasura.io/blog/with-hasura-celebrate-open-source-in-style-with-hacktoberfest/).
 
+#### **hacksquad**
+
+- **Swag**: T-shirts, hacksquad printed cups,carry bags,stickers,Hoodie
+- **Requirements**:  Register to the HackSquad using your GitHub, Join a squad or get assigned to a random squad, contribute code and get Swag!
```

Comment:
Don't just copy and place, please. And again, check punctuation and spelling, please.